index.js

```diff
@@ -22,6 +22,18 @@ MongoClient.connect(url, function(err, client) {
 app.use(router);
 app.use(morgan('combined'));
 
+app.post('/create-recipe', (req,res) => {
```

Comment:
```suggestion
app.post('/recipe', (req,res) => {
```
Dans une API REST, tes routes vont faire référence à des "ressources", c'est à dire l'objet qu'elles manipulent.
Dans ton cas la route POST sur `/recipe` aura pour but de créer une recette, la route GET sur `/recipe` aura pour but de lister l'ensemble des recettes
On ne met aps de verbe dans la route, il est uniquemeent dans la méthode http